POC/GPT4V-OCR.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Oh, ok. So they are JPEG. You'll have to pre-process the images and scale them down. Write a short script that runs once to scale the images and maybe appends '_scaled' to the file name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-herbarium/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Will work on this.</p>